vyper/parser/function_definitions/parse_function.py

```diff
@@ -1,11 +1,11 @@
+from vyper.parser import memory_allocator as mem_alloc
```

Comment:
why not import the class?

Reply:
`from [module] import [object]` doesn't work with mocking :(

Reply:
It breaks mocking.  I added a comment to explain.